pkg/iac-providers/kustomize/v3/load-dir_test.go

```diff
@@ -1,98 +1,87 @@
 package kustomizev3
 
 import (
-	"fmt"
-	"os"
 	"reflect"
-	"syscall"
 	"testing"
 
 	"github.com/accurics/terrascan/pkg/iac-providers/output"
 	"github.com/accurics/terrascan/pkg/utils"
 )
 
-var errorReadKustomize = fmt.Errorf("unable to read the kustomization file in the directory : %s", utils.ErrYamlFileEmpty.Error())
-
 func TestLoadIacDir(t *testing.T) {
 
 	table := []struct {
 		name          string
 		dirPath       string
 		kustomize     KustomizeV3
 		want          output.AllResourceConfigs
-		wantErr       error
```

Comment:
same thing.

Reply:
This change was made because we do not want a generic error to be propagated back to the caller functions.